Fix JUCE 7 + Clang: array parameter decays to pointer and no longer
matches AudioProcessor's initializer_list constructor.

--- a/libs/JUCE/modules/juce_audio_processors/processors/juce_AudioPluginInstance.h
+++ b/libs/JUCE/modules/juce_audio_processors/processors/juce_AudioPluginInstance.h
@@ -160,8 +160,8 @@
 
     AudioPluginInstance() = default;
     AudioPluginInstance (const BusesProperties& ioLayouts) : AudioProcessor (ioLayouts) {}
-    template <size_t numLayouts>
-    AudioPluginInstance (const short channelLayoutList[numLayouts][2]) : AudioProcessor (channelLayoutList) {}
+    AudioPluginInstance (const std::initializer_list<const short[2]>& channelLayoutList)
+        : AudioProcessor (channelLayoutList) {}
 
 private:
     // It's not safe to add a plain AudioProcessorParameter to an AudioPluginInstance.
